src/test/java/bio/terra/service/filedata/azure/tables/TableServiceClientUtilsTest.java

```diff
@@ -0,0 +1,129 @@
+package bio.terra.service.filedata.azure.tables;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.app.configuration.ConnectedTestConfiguration;
+import bio.terra.common.AzureUtils;
+import bio.terra.common.SynapseUtils;
+import bio.terra.common.category.Connected;
+import bio.terra.common.fixtures.ConnectedOperations;
+import bio.terra.common.fixtures.Names;
+import bio.terra.service.dataset.DatasetService;
+import bio.terra.service.filedata.FileService;
+import bio.terra.service.filedata.azure.AzureSynapsePdao;
+import bio.terra.service.filedata.azure.blobstore.AzureBlobStorePdao;
+import bio.terra.service.iam.IamProviderInterface;
+import bio.terra.service.resourcemanagement.azure.AzureAuthService;
+import com.azure.core.credential.AzureNamedKeyCredential;
+import com.azure.data.tables.TableClient;
+import com.azure.data.tables.TableServiceClient;
+import com.azure.data.tables.TableServiceClientBuilder;
+import com.azure.data.tables.models.TableEntity;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class TableServiceClientUtilsTest {
+  private static final Logger logger = LoggerFactory.getLogger(TableServiceClientUtilsTest.class);
+  private TableServiceClient tableServiceClient;
+  private String tableName;
+
+  @Autowired AzureSynapsePdao azureSynapsePdao;
+  @Autowired ConnectedOperations connectedOperations;
+  @Autowired private ConnectedTestConfiguration testConfig;
+  @Autowired DatasetService datasetService;
+  @MockBean private IamProviderInterface samService;
+  @Autowired SynapseUtils synapseUtils;
+  @Autowired AzureAuthService azureAuthService;
+  @Autowired TableDirectoryDao tableDirectoryDao;
+  @Autowired TableDao tableDao;
+  @Autowired AzureBlobStorePdao azureBlobStorePdao;
+  @Autowired FileService fileService;
+  @Autowired AzureUtils azureUtils;
```

Comment:
```suggestion
  @Autowired ConnectedOperations connectedOperations;
  @Autowired private ConnectedTestConfiguration testConfig;
  @MockBean private IamProviderInterface samService;
  @Autowired AzureUtils azureUtils;
```

There are a couple of autowired components here that aren't being used